### ISSUE TITLE
HTTP 404 Not Found on service deploy

### ISSUE THREAD

**Issue by SpComb** [kontena/kontena#1454]
Appears to be related to deployment failures.

`$ DEBUG=true  kontena app deploy`
```
...
    *   Deploying lb  .. -D, [2016-11-30T15:08:51.333965 #22659] DEBUG -- CLIENT: Request GET /v1/services/tsystems/null/tss-registry-lb/deploys/583edd72c8e86e00060017a8: 404 Not Found: {"error":"Not found"}
 [fail] Deploying lb
Not found Not found
/home/kraken/.gem/ruby/2.3.0/gems/kontena-cli-1.0.0/lib/kontena/client.rb:512:in `handle_error_response'
/home/kraken/.gem/ruby/2.3.0/gems/kontena-cli-1.0.0/lib/kontena/client.rb:342:in `rescue in request'
/home/kraken/.gem/ruby/2.3.0/gems/kontena-cli-1.0.0/lib/kontena/client.rb:283:in `request'
/home/kraken/.gem/ruby/2.3.0/gems/kontena-cli-1.0.0/lib/kontena/client.rb:196:in `get'
/home/kraken/.gem/ruby/2.3.0/gems/kontena-cli-1.0.0/lib/kontena/cli/services/services_helper.rb:231:in `block in wait_for_deploy_to_finish'
/usr/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
/usr/lib/ruby/2.3.0/timeout.rb:33:in `block in catch'
/usr/lib/ruby/2.3.0/timeout.rb:33:in `catch'
/usr/lib/ruby/2.3.0/timeout.rb:33:in `catch'
/usr/lib/ruby/2.3.0/timeout.rb:106:in `timeout'
/home/kraken/.gem/ruby/2.3.0/gems/kontena-cli-1.0.0/lib/kontena/cli/services/services_helper.rb:229:in `wait_for_deploy_to_finish'
/home/kraken/.gem/ruby/2.3.0/gems/kontena-cli-1.0.0/lib/kontena/cli/apps/deploy_command.rb:56:in `block (2 levels) in deploy_services'
/home/kraken/.gem/ruby/2.3.0/gems/kontena-cli-1.0.0/lib/kontena/cli/spinner.rb:124:in `spin'
/home/kraken/.gem/ruby/2.3.0/gems/kontena-cli-1.0.0/lib/kontena/cli/common.rb:278:in `spinner'
/home/kraken/.gem/ruby/2.3.0/gems/kontena-cli-1.0.0/lib/kontena/cli/apps/deploy_command.rb:53:in `block in deploy_services'
/home/kraken/.gem/ruby/2.3.0/gems/kontena-cli-1.0.0/lib/kontena/cli/apps/deploy_command.rb:46:in `each'
/home/kraken/.gem/ruby/2.3.0/gems/kontena-cli-1.0.0/lib/kontena/cli/apps/deploy_command.rb:46:in `deploy_services'
/home/kraken/.gem/ruby/2.3.0/gems/kontena-cli-1.0.0/lib/kontena/cli/apps/deploy_command.rb:32:in `execute'
/home/kraken/.gem/ruby/2.3.0/gems/kontena-cli-1.0.0/lib/kontena/command.rb:187:in `run'
/home/kraken/.gem/ruby/2.3.0/gems/clamp-1.1.1/lib/clamp/subcommand/execution.rb:11:in `execute'
/home/kraken/.gem/ruby/2.3.0/gems/kontena-cli-1.0.0/lib/kontena/command.rb:187:in `run'
/home/kraken/.gem/ruby/2.3.0/gems/clamp-1.1.1/lib/clamp/subcommand/execution.rb:11:in `execute'
/home/kraken/.gem/ruby/2.3.0/gems/kontena-cli-1.0.0/lib/kontena/command.rb:187:in `run'
/home/kraken/.gem/ruby/2.3.0/gems/clamp-1.1.1/lib/clamp/command.rb:132:in `run'
/home/kraken/.gem/ruby/2.3.0/gems/kontena-cli-1.0.0/bin/kontena:16:in `<top (required)>'
/home/kraken/.local/bin/kontena:23:in `load'
/home/kraken/.local/bin/kontena:23:in `<main>'
```

**Comment by SpComb** [comment]
Fixed by  #1523 for the case of `kontena service deploy` while the service balancer or another deploy is still active.

**Comment by SpComb** [comment]
I think interactions between `kontena service stop` and `kontena service deploy` can still trigger this?
